src/components/AddContactDialog.js

```diff
@@ -106,7 +108,7 @@ class AddContactDialog extends React.Component {
       </Dialog>
     );
   }
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
@nithyanatarajan kindly use the appropriate life cycle event instead of using deprecated life cycle event